great_expectations_provider/example_dags/example_great_expectations_dag.py

```diff
@@ -3,113 +3,73 @@
 """
 
 import os
-import airflow
+from datetime import datetime
+from pathlib import Path
+
 from airflow import DAG
-from great_expectations_provider.operators.great_expectations import (
-    GreatExpectationsOperator,
-    GreatExpectationsBigQueryOperator,
-)
 
-default_args = {"owner": "Airflow", "start_date": airflow.utils.dates.days_ago(1)}
+from great_expectations_provider.operators.great_expectations import GreatExpectationsOperator
 
-dag = DAG(dag_id="example_great_expectations_dag", default_args=default_args)
+from include.object_configs.example_data_context_config import example_data_context_config
```

Comment:
Would be helpful to have these under the Great Expectations folder in `include/`, that way users' `include/` folders can stay more organized with the provider files all under the same space.